a/staging/src/kubevirt.io/api/core/v1/types_swagger_generated.go b/staging/src/kubevirt.io/api/core/v1/types_swagger_generated.go
index 213228f96..ff3fbadb4 100644
--- a/staging/src/kubevirt.io/api/core/v1/types_swagger_generated.go
+++ b/staging/src/kubevirt.io/api/core/v1/types_swagger_generated.go
@@ -72,6 +72,7 @@ func (VirtualMachineInstanceStatus) SwaggerDoc() map[string]string {
 		"evacuationNodeName":            "EvacuationNodeName is used to track the eviction process of a VMI. It stores the name of the node that we want\nto evacuate. It is meant to be used by KubeVirt core components only and can't be set or modified by users.\n+optional",
 		"activePods":                    "ActivePods is a mapping of pod UID to node name.\nIt is possible for multiple pods to be running for a single VMI during migration.",
 		"volumeStatus":                  "VolumeStatus contains the statuses of all the volumes\n+optional\n+listType=atomic",
+		"kernelBootStatus":              "KernelBootStatus contains info about the kernelBootContainer\n+optional",
 		"fsFreezeStatus":                "FSFreezeStatus is the state of the fs of the guest\nit can be either frozen or thawed\n+optional",
 		"topologyHints":                 "+optional",
 		"virtualMachineRevisionName":    "VirtualMachineRevisionName is used to get the vm revision of the vmi when doing\nan online vm snapshot\n+optional",
@@ -108,6 +109,29 @@ func (VolumeStatus) SwaggerDoc() map[string]string {
 		"hotplugVolume":             "If the volume is hotplug, this will contain the hotplug status.",
 		"size":                      "Represents the size of the volume",
 		"memoryDumpVolume":          "If the volume is memorydump volume, this will contain the memorydump info.",
+		"containerDiskVolume":       "ContainerDiskVolume shows info about the containerdisk, if the volume is a containerdisk",
+	}
+}
+
+func (KernelInfo) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "KernelInfo show info about the kernel image",
+		"checksum": "Checksum is the checksum of the kernel image",
+	}
+}
+
+func (InitrdInfo) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "InitrdInfo show info about the initrd file",
+		"checksum": "Checksum is the checksum of the initrd file",
+	}
+}
+
+func (KernelBootStatus) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":           "KernelBootStatus contains info about the kernelBootContainer",
+		"kernelInfo": "KernelInfo show info about the kernel image",
+		"initrdInfo": "InitrdInfo show info about the initrd file",
 	}
 }
 
@@ -129,6 +153,13 @@ func (HotplugVolumeStatus) SwaggerDoc() map[string]string {
 	}
 }
 
+func (ContainerDiskInfo) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "ContainerDiskInfo shows info about the containerdisk",
+		"checksum": "Checksum is the checksum of the rootdisk or kernel artifacts inside the containerdisk",
+	}
+}
+
 func (VirtualMachineInstanceCondition) SwaggerDoc() map[string]string {
 	return map[string]string{
 		"lastProbeTime":      "+nullable",
